packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/Drawing/DrawingContainer.js

```diff
@@ -35,70 +38,58 @@ class DrawingContainer extends Component {
   componentDidMount () {
     this.setActiveMark()
 
-    // TEMP CODE UNTIL COORDINATE STREAM
-    const { eventStream } = this.props
-    if (eventStream) {
-      this.tempSubscription = eventStream.subscribe(event => {
-        if (event.type === 'pointerup') {
-          this.tempMarkCreation()
-        }
-      })
-    }
+    // TEMP CODE UNTIL TOOL STORES
+    const { coordinateStream } = this.props
+    this.tempSubscription = coordinateStream.subscribe(event => {
+      if (event.type === 'pointerup') {
+        this.tempMarkCreation()
+      }
+    })
   }
 
   componentDidUpdate (prevProps) {
-    if (this.props.activeStepTasks !== prevProps.activeStepTasks || this.props.activeDrawingTool !== prevProps.activeDrawingTool) {
+    if (this.props.activeDrawingTask !== prevProps.activeDrawingTask || this.props.activeDrawingTool !== prevProps.activeDrawingTool) {
```

Comment:
Is something missing from this `||` statement? Both conditions appear to be identical.

Reply:
`activeDrawingTool` is the array index for the selected drawing task tool, while the `activeDrawingTask` is the task object including tools, instructions, help, etc.